src/sync/installation.ts

```diff
@@ -209,14 +209,221 @@ export const isNotFoundError = (
 	return isNotFoundError;
 };
 
+// TODO: type queues
+async function doProcessInstallation(app, queues, job, installationId: number, jiraHost: string, logger: LoggerWithTarget): Promise<void> {
+	const subscription = await Subscription.getSingleInstallation(
+		jiraHost,
+		installationId
+	);
+	// TODO: should this reject instead? it's just ignoring an error
```

Comment:
If you haven't already, can you create a ticket for this to make sure we follow up on it?

Reply:
Yeah, killing it here https://softwareteams.atlassian.net/browse/ARC-764 in my next PR